test/browser/pages/ipvError.js

```diff
@@ -0,0 +1,32 @@
+module.exports = class PlaywrightDevPage {
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+  constructor(page) {
+    this.page = page;
+    this.path = "/redirect";
+  }
+
+  getErrorText() {
+    return this.page.textContent("body > pre");
```

Comment:
Is there an `id` or `data-test` attribute we could use instead? I'm not even sure what `body > pre` is or how likely it is to change